Installation blocked by -D option on Monterey Mac

Hello! I am having trouble finishing installation.

```bash
for dirname in $(find src -type d | sed 's/'src'/'bin'/'); do mkdir -p $dirname; done
mkdir -p ~/.local/share/kabmat
touch ~/.local/share/kabmat/data
/Library/Developer/CommandLineTools/usr/bin/make kabmat
c++ -std=c++17 -Wall -Wextra -c src/ArgsParser/ArgsParser.cpp -o bin/ArgsParser/ArgsParser.o -lncurses
clang: warning: -lncurses: 'linker' input unused [-Wunused-command-line-argument]
c++ -std=c++17 -Wall -Wextra -c src/ui/screens/Board/Board.cpp -o bin/ui/screens/Board/Board.o -lncurses
clang: warning: -lncurses: 'linker' input unused [-Wunused-command-line-argument]
c++ -std=c++17 -Wall -Wextra -c src/ui/screens/MainMenu/MainMenu.cpp -o bin/ui/screens/MainMenu/MainMenu.o -lncurses
clang: warning: -lncurses: 'linker' input unused [-Wunused-command-line-argument]
c++ -std=c++17 -Wall -Wextra -c src/ui/components/Footer/Footer.cpp -o bin/ui/components/Footer/Footer.o -lncurses
clang: warning: -lncurses: 'linker' input unused [-Wunused-command-line-argument]
c++ -std=c++17 -Wall -Wextra -c src/ui/components/Input/Input.cpp -o bin/ui/components/Input/Input.o -lncurses
clang: warning: -lncurses: 'linker' input unused [-Wunused-command-line-argument]
c++ -std=c++17 -Wall -Wextra -c src/ui/components/ConfirmDialog/ConfirmDialog.cpp -o bin/ui/components/ConfirmDialog/ConfirmDialog.o -lncurses
clang: warning: -lncurses: 'linker' input unused [-Wunused-command-line-argument]
c++ -std=c++17 -Wall -Wextra -c src/ui/components/CardWin/CardWin.cpp -o bin/ui/components/CardWin/CardWin.o -lncurses
clang: warning: -lncurses: 'linker' input unused [-Wunused-command-line-argument]
c++ -std=c++17 -Wall -Wextra -c src/ui/components/ColumnWin/ColumnWin.cpp -o bin/ui/components/ColumnWin/ColumnWin.o -lncurses
clang: warning: -lncurses: 'linker' input unused [-Wunused-command-line-argument]
c++ -std=c++17 -Wall -Wextra -c src/ui/components/CardInfo/CardInfo.cpp -o bin/ui/components/CardInfo/CardInfo.o -lncurses
clang: warning: -lncurses: 'linker' input unused [-Wunused-command-line-argument]
c++ -std=c++17 -Wall -Wextra -c src/ui/components/Header/Header.cpp -o bin/ui/components/Header/Header.o -lncurses
clang: warning: -lncurses: 'linker' input unused [-Wunused-command-line-argument]
c++ -std=c++17 -Wall -Wextra -c src/ui/components/Help/Help.cpp -o bin/ui/components/Help/Help.o -lncurses
clang: warning: -lncurses: 'linker' input unused [-Wunused-command-line-argument]
c++ -std=c++17 -Wall -Wextra -c src/ui/components/Checklist/Checklist.cpp -o bin/ui/components/Checklist/Checklist.o -lncurses
clang: warning: -lncurses: 'linker' input unused [-Wunused-command-line-argument]
c++ -std=c++17 -Wall -Wextra -c src/ui/helpers/win_fit_text/win_fit_text.cpp -o bin/ui/helpers/win_fit_text/win_fit_text.o -lncurses
clang: warning: -lncurses: 'linker' input unused [-Wunused-command-line-argument]
c++ -std=c++17 -Wall -Wextra -c src/ui/helpers/win_center_text/win_center_text.cpp -o bin/ui/helpers/win_center_text/win_center_text.o -lncurses
clang: warning: -lncurses: 'linker' input unused [-Wunused-command-line-argument]
c++ -std=c++17 -Wall -Wextra -c src/DataManager/Column.cpp -o bin/DataManager/Column.o -lncurses
clang: warning: -lncurses: 'linker' input unused [-Wunused-command-line-argument]
c++ -std=c++17 -Wall -Wextra -c src/DataManager/DataManager.cpp -o bin/DataManager/DataManager.o -lncurses
clang: warning: -lncurses: 'linker' input unused [-Wunused-command-line-argument]
c++ -std=c++17 -Wall -Wextra -c src/DataManager/Board.cpp -o bin/DataManager/Board.o -lncurses
clang: warning: -lncurses: 'linker' input unused [-Wunused-command-line-argument]
c++ -std=c++17 -Wall -Wextra -c src/DataManager/Card.cpp -o bin/DataManager/Card.o -lncurses
clang: warning: -lncurses: 'linker' input unused [-Wunused-command-line-argument]
c++ -std=c++17 -Wall -Wextra -c src/helpers/trim_spaces/trim_spaces.cpp -o bin/helpers/trim_spaces/trim_spaces.o -lncurses
clang: warning: -lncurses: 'linker' input unused [-Wunused-command-line-argument]
c++ -std=c++17 -Wall -Wextra -c src/main.cpp -o bin/main.o -lncurses
clang: warning: -lncurses: 'linker' input unused [-Wunused-command-line-argument]
c++  bin/ArgsParser/ArgsParser.o  bin/ui/screens/Board/Board.o  bin/ui/screens/MainMenu/MainMenu.o  bin/ui/components/Footer/Footer.o  bin/ui/components/Input/Input.o  bin/ui/components/ConfirmDialog/ConfirmDialog.o  bin/ui/components/CardWin/CardWin.o  bin/ui/components/ColumnWin/ColumnWin.o  bin/ui/components/CardInfo/CardInfo.o  bin/ui/components/Header/Header.o  bin/ui/components/Help/Help.o  bin/ui/components/Checklist/Checklist.o  bin/ui/helpers/win_fit_text/win_fit_text.o  bin/ui/helpers/win_center_text/win_center_text.o  bin/DataManager/Column.o  bin/DataManager/DataManager.o  bin/DataManager/Board.o  bin/DataManager/Card.o  bin/helpers/trim_spaces/trim_spaces.o  bin/main.o -std=c++17 -Wall -Wextra -o kabmat -lncurses
❯ sudo make
for dirname in $(find src -type d | sed 's/'src'/'bin'/'); do mkdir -p $dirname; done
mkdir -p ~/.local/share/kabmat
touch ~/.local/share/kabmat/data
/Library/Developer/CommandLineTools/usr/bin/make kabmat
make[1]: `kabmat' is up to date.
❯ sudo make install
install -dm755 /usr/local/bin /usr/local/share/man/man1
install -Dm755 ./kabmat /usr/local/bin
install: illegal option -- D
usage: install [-bCcpSsv] [-B suffix] [-f flags] [-g group] [-m mode]
               [-o owner] file1 file2
       install [-bCcpSsv] [-B suffix] [-f flags] [-g group] [-m mode]
               [-o owner] file1 ... fileN directory
       install -d [-v] [-g group] [-m mode] [-o owner] directory ...
make: *** [install] Error 64
```

After running `make install --help`, I get:

```bash
❯ make install --help
Usage: make [options] [target] ...
Options:
  -b, -m                      Ignored for compatibility.
  -B, --always-make           Unconditionally make all targets.
  -C DIRECTORY, --directory=DIRECTORY
                              Change to DIRECTORY before doing anything.
  -d                          Print lots of debugging information.
  --debug[=FLAGS]             Print various types of debugging information.
  -e, --environment-overrides
                              Environment variables override makefiles.
  -f FILE, --file=FILE, --makefile=FILE
                              Read FILE as a makefile.
  -h, --help                  Print this message and exit.
  -i, --ignore-errors         Ignore errors from commands.
  -I DIRECTORY, --include-dir=DIRECTORY
                              Search DIRECTORY for included makefiles.
  -j [N], --jobs[=N]          Allow N jobs at once; infinite jobs with no arg.
  -k, --keep-going            Keep going when some targets can't be made.
  -l [N], --load-average[=N], --max-load[=N]
                              Don't start multiple jobs unless load is below N.
  -L, --check-symlink-times   Use the latest mtime between symlinks and target.
  -n, --just-print, --dry-run, --recon
                              Don't actually run any commands; just print them.
  -o FILE, --old-file=FILE, --assume-old=FILE
                              Consider FILE to be very old and don't remake it.
  -p, --print-data-base       Print make's internal database.
  -q, --question              Run no commands; exit status says if up to date.
  -r, --no-builtin-rules      Disable the built-in implicit rules.
  -R, --no-builtin-variables  Disable the built-in variable settings.
  -s, --silent, --quiet       Don't echo commands.
  -S, --no-keep-going, --stop
                              Turns off -k.
  -t, --touch                 Touch targets instead of remaking them.
  -v, --version               Print the version number of make and exit.
  -w, --print-directory       Print the current directory.
  --no-print-directory        Turn off -w, even if it was turned on implicitly.
  -W FILE, --what-if=FILE, --new-file=FILE, --assume-new=FILE
                              Consider FILE to be infinitely new.
  --warn-undefined-variables  Warn when an undefined variable is referenced.
  -N OPTION, --NeXT-option=OPTION
                              Turn on value of NeXT OPTION.

This program built for i386-apple-darwin11.3.0
Report bugs to <bug-make@gnu.org>
```

So it looks like I'm missing the directory options needed to finish this, and I'm not sure replacing with `C` in my case will work, since it looks like this takes more than just changing directories. Please advise!

I am running `zsh` and `macOS Monterey 12.2.1`.

--- a/Makefile
+++ b/Makefile
@@ -42,9 +42,11 @@ clean:
 
 .PHONY: install
 install:
-	install -dm755 $(DESTDIR)$(BINDIR) $(DESTDIR)$(MANDIR)/man1
-	install -Dm755 ./$(TARGET) $(DESTDIR)$(BINDIR)
-	install -Dm644 ./doc/kabmat.1 $(DESTDIR)$(MANDIR)/man1
+	mkdir -p $(DESTDIR)$(BINDIR) $(DESTDIR)$(MANDIR)/man1
+	cp ./$(TARGET) $(DESTDIR)$(BINDIR)
+	chmod 0755 $(DESTDIR)$(BINDIR)/$(TARGET)
+	cp ./doc/kabmat.1 $(DESTDIR)$(MANDIR)/man1
+	chmod 0644 $(DESTDIR)$(MANDIR)/man1/kabmat.1
 	$(MAKE) clean
 
 .PHONY: uninstall

